sips/minutes/2016-10-25-sip-minutes.md

```diff
@@ -0,0 +1,173 @@
+thinks that there should be a fixed deadline for voting.
+
+All the Committee agrees that with the new rules the Committee wouldn't vote
+publicly, because Committee members can notify their vote after the meeting and
+before the fixed deadline.
+
+Martin proposes to keep the old rules, the 70% percentage rule. Jorge proposes
+to have simpler and more predictable rules, and replace 70% with the two-thirds
+rule. Martin comes back to the abstentions point: he doesn't want them to be
+considered as no, because with the current rules Committee members could block
+the whole decision process. Some Committee members point out that this wouldn't
+happen because then the quorum wouldn't be reached. The discussion shifts
+towards the quorum point: is it necessary anymore? Heather and Adriaan say that
+it is, because it ensures that a reasonable number of Committee members meet to
+discuss proposals and convince each other about the pros and cons. All the
+Committee finally decides to not consider abstentions as no's. Adriaan, however,
+suggests that we should penalize people that abstain too often.
+
+Another system proposed by Martin helps survive the abstention issue mentioned
+before. For a proposal to pass, at least 50% of all the Committee 
+and two-thirds of all the people voting (not counting abstentions) must accept
+it. The quorum rule is kept. After some discussion, the Committee unanimously
+votes in favor of the new rules.
+
+### Discussion of SIP-27: Trailing Commas
+
+Eugene describes what happened in the last meeting. Martin asks if there is a
+concrete proposal. Jorge says that he thinks that Dale would push for the new
+line specialized case. Martin proposes to push for a more concrete proposal
+because the specification cannot be so open-ended to properly analyze the
+consequences of the suggested changes.
+
+Seth and Adriaan voice their opinions on trailing commas again, they don't agree
+this is a problem that should be solved in the language. Martin changes his view
+on the subject, and provided that the proposal does no harm to the syntax of the
+language and doesn't cause problems, he would accept the change. He also says
+that it wouldn't be good to contradict last month's vote on the proposal.
+Heather says that she's in favor of trailing commas because makes beginners be
+less confused by the syntax and help people get started with the language.
+Iulian doesn't like the specialized version, he would prefer a general version.
+
+**Outcome**: All the Committee decides to vote on accepting the proposal again for a final
+review with a clear specification of the changes. Seth and Adriaan vote against,
+the rest of the Committee except Josh vote in favor of it. According to our new
+rules, Josh gets to decide whether the proposal will be readmitted for review or
+cancelled (**NOTE**: Josh votes in favor of trailing commas one week after the
+meeting). Trailing commas will be reviewed in November.
```

Comment:
No, not yet. Dale didn't decide on a concrete change of the syntax, so Martin wants to check the final spec and see its implications. We're voting on it this month (29th November) since Dale has already submitted the changes and decided for one of the three options he proposed in its last specification.